code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -106,6 +106,12 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 	var/cached_gases = gases
 	TOTAL_MOLES(cached_gases, .)
 
+	/// Checks to see if gas amount exists in mixture
```

Comment:
```suggestion
/// Checks to see if gas amount exists in mixture
```

Reply:
All of the documentation for the procs have 1 tab in front of them.

Reply:
Yes, and it's wrong

Reply:
Dennok did me dirty when he documented these

Reply:
I see.  Well it won't be much hassle to remove all the tabs from the file for the documentation.

Reply:
228b21d removes all the whitespace from the file.